ztrouy/cpyou
# Feature - Polish UX

### Purpose
To improve the User experience by stabilizing the placement of UI components in the Build Form view, adding a favicon for Users to more easily identify the website's tab, and add a "Back" button on the Build Details view.

### Files Changed
- Added calculated property "FullName" to `MemoryNoNavDTO.cs`, `MemoryForBuildDetailsDTO.cs`, BuildMemoryForEditFormDTO.cs`, `StorageNoNavDTO.cs`, `StorageForBuildDetailsDTO.cs`, and `BuildStorageForEditFormDTO.cs`
- Added favicon vector images in new `CPUCircle.svg` and `CPUCircleLarge.svg` vector files
- Updated favicon to use CPUCircleLarge.svg in `index.html`
- Updated Storage and Memory elements to use new FullName property, added a Back button that navigates to Builds List view, and removed unused imports in `BuildDetails.jsx`
- Updated rendering of chosen Memory and Storage devices to be more stable in position, and to use new calculated FullName property in `BuildForm.jsx`

### To Test
Fetch, setup, and run according to project README, then confirm the following
- Favicon for website is CPYou logo in a deep blue circle
- Build Details view has a back button that navigates user to the Builds List view
- Quantity selector for Storage and Memory devices in the Build Form view does not get pushed to the left when its price reaches or exceeds $1000.00

closes #50
